src/goto-analyzer/goto_analyzer_parse_options.h

```diff
@@ -129,7 +130,17 @@ class optionst;
   "(intervals)" \
   "(non-null)" \
   "(constants)" \
-  "(dependence-graph)"
+  "(dependence-graph)" \
```

Comment:
💡 `dependency-graph` seems to be more common.

Reply:
Yes but this is what the class in question is called and it has been a command-line option for some time.  I'm happy to change it but I don't think it should be part of this PR.